assets/images/Leanne Gallagher _ Portfolio.html

```diff
@@ -0,0 +1,198 @@
+
+  <body>
+    <header class="header">
+      <img src="./Leanne Gallagher _ Portfolio_files/profile-img.png">
+      <h1>Leanne Gallagher</h1>
+      <!-- The navigation bar containing internal links to each important section of the webpage -->
+      <nav class="nav-container">
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#about">About Me</a>
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#portfolio">Portfolio</a>
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#contact">Contact</a>
+      </nav>
+    </header>
+
+    <main>
+      <!-- First main section of webpage with a short description about myself -->
+      <section id="about" class="about-me">
+        <h1>About Me</h1>
+        <p>
+          Hi, I'm Leanne. I'm 29 years old and I'm from Coventry. I am just
+          beginning my journey in coding and I am looking forward to where this
+          journey may take me. I am hardworking, inquisitive and passionate to
+          learn all I can.
+        </p>
+      </section>
+
+      <!-- Second section containing the portfolio and heading-->
+      <section id="portfolio" class="portfolio">
+        <h1>Portfolio</h1>
+
+        <!-- This is the section for the first project in the portfolio that has to span across the webpage -->
+        <section class="main-project-card">
+          <img src="./Leanne Gallagher _ Portfolio_files/main-project-screenshot.png">
+          <!-- This contains the information related to the first project including a short description and external link -->
+          <div class="main-project-card-info">
+            <h3>Semantic HTML Refactor</h3>
+            <p>
+              The objective of this project was to fix broken code, make the
+              webpage more accessible and refactor HTML semantics.
+            </p>
+            <a href="https://lenny-g.github.io/semantic-html-refactor/">view</a>
+          </div>
+        </section>
+        <!-- This is a section contained in the Portfolio section that holds the 6 other project cards and their placeholders to be edited at a later date -->
+        <section class="other-projects">
+          <div class="project-card">
+            <img src="./Leanne Gallagher _ Portfolio_files/250x250">
+            <div class="card-container">
+              <h4>Project 2</h4>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque
+                beatae aliquid sit a architecto maiores doloribus odio et
+                perspiciatis corrupti, fugiat mollitia. Aperiam iure laudantium
+                harum facere beatae quod culpa.
+              </p>
```

Comment:
Same as above with Lorem Ipsum. Consider just narrowing this down to 'Coming soon!' or something similar.